tests/Algorithm/test_evaluation_module.py

```diff
@@ -46,6 +46,7 @@ def test_givenOnePlayerInMyTeamFarFromGoal_thenLineOfSightClearanceIsDistanceToT
 
         assert line_of_sight_clearance(player1, self.goal) == distance_to_target
 
+    @unittest.skip
     def test_givenOnePlayerInMyTeamNearFromGoal_thenLineOfSightClearanceIsDistanceToTargetTimesPathScore(self):
```

Comment:
What is making this test fail?

Reply:
available_player output no player, because there is no avaible player in the test...

Reply:
```python
GameState().our_team.available_players.values()
```

Reply:
wtf now they worked